src/content/web3/index.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Web3
+description: What is web3 and why is it important?
+lang: en
+sidebar: true
+---
+
+# Introduction to web3 {#introduction}
+
+Centralization has helped onboard billions of people to the internet and created the stable, robust infrastructure on which it lives. At the same time, these centralized entities have a stronghold on large swathes of the internet, undemocratically deciding what should and should not be allowed.
```

Comment:
> deciding what should and should not be allowed.

This also feels a bit vague. "Allowed" in what sense? What is allowed to exist on the internet?

Perhaps we could be a bit more specific about how they decide:
- Who can participate
- How people participate
- Overall, how internet users consume, create & interact with information